gui/src/installer/view.rs

```diff
@@ -26,7 +26,9 @@ use crate::{
     installer::{
         context::Context,
         message::{self, Message},
-        prompt, Error,
+        prompt,
+        step::{DownloadState, InstallState},
```

Comment:
Generally, I prefer to not have circular dependencies. Can you cut the view in multiple functions and call in the step method view the one according to the current step state ? Or would you prefer we do it in other PRs ?

Reply:
Ah yes, that's a good point. I'll have a look now at how best to split it and if I can do it quickly, but otherwise it might be better in a follow-up PR in order not to delay the release.

Reply:
Would it be OK to define parts of the view within the step as an `Element` and pass this to the view function? That way I could `match` on different values of `DownloadState` and `InstallState` within the step and avoid the need to define different functions for each case.

Reply:
In a followed up PR I would prefer finally